generator/src/main/java/com/linkedin/pegasus/generator/DataSchemaParser.java

```diff
@@ -191,6 +209,19 @@ private static ParseResult combine(Collection<ParseResult> parseResults)
       return _schemaAndLocations;
     }
 
+    public Map<DataSchema, DataSchemaLocation> getExtensionDataSchemaAndLocations()
+    {
+      return _schemaAndLocations.entrySet().stream().filter(entry -> {
+        DataSchemaLocation dataSchemaLocation = entry.getValue();
+        if (dataSchemaLocation instanceof InJarFileDataSchemaLocation)
+        {
+          InJarFileDataSchemaLocation inJarFileDataSchemaLocation = (InJarFileDataSchemaLocation) dataSchemaLocation;
+          return inJarFileDataSchemaLocation.getPathInJar().startsWith("extension");
```

Comment:
First of all, it is "extensions". Second of all, can we make it as constant? or just use SchemaDirectoryName.EXTENSIONS.getName().

Reply:
good suggestion